wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/WorkspaceManager.java

```diff
@@ -58,9 +58,15 @@
   private static final Logger LOG = LoggerFactory.getLogger(WorkspaceManager.class);
 
   /** This attribute describes time when workspace was created. */
-  public static final String CREATED_ATTRIBUTE_NAME = "created";
-  /** This attribute describes time when workspace was last updated or started/stopped/recovered. */
-  public static final String UPDATED_ATTRIBUTE_NAME = "updated";
+  protected static final String CREATED_ATTRIBUTE_NAME = "created";
```

Comment:
I think such constant should be placed in `org.eclipse.che.api.workspace.shared.Constants` class. Then [here](https://github.com/eclipse/che/pull/7988/files#diff-1340504bd34c91a6a4c1666d91511f01R192) you will be able to use constant instead of hardcoded string